backend/apps/core/tests/test_throttling.py

```
@@ -0,0 +1,238 @@
+        for i in range(100):
+            response = self.client.get('/api/v1/content-types/')
+            self.assertEqual(response.status_code, status.HTTP_200_OK, f"Request {i+1} failed")
+
+
+class LoginBurstProtectionTests(ThrottlingTestCase):
+    """Test burst protection on login endpoints."""
+
+    def test_login_burst_protection_10_per_minute(self):
+        """Test that login endpoint has burst protection (10/minute)."""
+        # Make 10 login attempts - should all be allowed (even if auth fails)
+        for i in range(10):
```

Ruff (0.14.3)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)